src/lang/Messages_de_DE.properties

```diff
@@ -302,6 +302,7 @@ ascan.toolbar.button.new=New Scan
 ascan.toolbar.button.stop=Aktiven Scan stoppen
 ascan.toolbar.button.unpause=Aktiven Scan fortsetzen
 ascan.toolbar.confirm.clear=Are you sure you want to clear all completed scans?
+ascan.toolbar.newissues.label=New issues\:
```

Comment:
Note that you just need to change the main Messages.properties file, the translation files are updated automatically when needed, by @zapbot :)